posts/2020-12-18-microprofile-long-running-actions-beta.adoc

```diff
@@ -176,26 +182,28 @@ Finally the compensate method looks like this:
         return Response.ok(ParticipantStatus.Compensated.name()).build();
     }
 ----
-Very similar to the complete method except with the `@Compensate` annotation. This will be called should any service in the LRA fail and so should include any business logic that will roll back any changes to the services data that had been made by the `@LRA` method and return it to its original state. It falls on the service developer to know how to roll this back and the LRA implementation plays no part in it but will ensure that the logic is run should the LRA fail.
+The compensate method is similar to the complete method, except it uses the `@Compensate` annotation. This method is called if any service in the LRA fails. It includes any business logic that is necessary to roll back changes that the `@LRA` method made and returning the service to its original state. It is up to the service developer to know how to roll the service back. The LRA implementation plays no part in the rollback except to ensure that the logic is run if the LRA fails.
 
-While these 3 annotations form the basics of a LRA there are several more that can be used
+While these three annotations form the basics of an LRA, there are several more annotations avaialble:
 
-* `@Forget` - Called if the complete or compensate methods failed and you want to release any resources allocated to the LRA
-* `@Leave` - Called if the class is no longer interested in the LRA
-* `@Status` - When invoked a method with this annotation will return the status of the LRA
-* `@AfterLRA` - When a LRA is in its final state, the method with this annotation is called
+* `@Forget` - called if the complete or compensate methods fail and you want to release any resources that were allocated to the LRA
+* `@Leave` - called if the class is no longer interested in the LRA
+* `@Status` - When a method with this annotation is invoked, it returns the status of the LRA.
```

Comment:
Sometimes added a `.` at the end of the point and other times not

Reply:
changed them all to be complete sentences that end with a period